apps/daimo-mobile/src/logic/daimoContacts.ts

```diff
@@ -142,11 +145,11 @@ export function useRecipientSearch(
   if (recipients.length === 0) recipients.push(...looseMatchRecents);
 
   // If we have a valid phone number or email prefix, show it as a recipient
-  if (zEmailAddress.safeParse(prefix).success) {
+  if (zEmailAddress.safeParse(prefix).success && !onlyDaimoContacts) {
```

Comment:
we *can* send requests to email, phone, system contacts. it'll send as a link

we just can't send them to non-Daimo Ethereum accounts, since we have no way to message those

Reply:
Yeah. In those cases, the user can probably use the "Request via link" button. This filters the search results to not include email or phone contacts, since those would lead to the same screen that the "Request via link" button would.

Reply:
this is true of the send flow too, right?

(sending to Alice, a phone contact) vs (Send to Link) both creates a link, first one is just a nice affordance to pre-select where you're going to send the link afterward

i think request could work same way